packages/functional-tests/tests/signin/relyingParties.spec.ts

```diff
@@ -10,6 +10,7 @@ test.describe('severity-1 #smoke', () => {
     target,
     credentials,
   }) => {
+    test.slow();
```

Comment:
Why are we adding `test.slow()` here?

Reply:
Last few times I'd run the tests, this one was timing out consistently without `test.slow()`. Tried again today though and no longer reproducing.

Reply:
As discussed on Slack, the timeouts started occurring again with inconsistent failures on different tests because of issues with credentials teardown. Added a `test.slow()` at the describe level and seeing tests now consistently passing.

Reply:
fun fact: we currently have 62 `test.slow()` calls in functional tests (7 of which seem to be conditional on environment due to potential slow email delivery in stage/prod).
```sh
git grep "test.slow(" -- "packages/functional-tests/**/*.spec.ts" | wc -l # 62
```

```sh
git grep -n "test.slow(" -- "packages/functional-tests/**/*.spec.ts" | grep -v "test.slow()"

packages/functional-tests/tests/oauth/forceAuth.spec.ts:34:      test.slow(project.name !== 'local', 'email delivery can be slow');
packages/functional-tests/tests/oauth/forceAuth.spec.ts:52:      test.slow(project.name !== 'local', 'email delivery can be slow');
packages/functional-tests/tests/oauth/signin.spec.ts:13:      test.slow(
packages/functional-tests/tests/oauth/signin.spec.ts:135:      test.slow(project.name !== 'local', 'email delivery can be slow');
packages/functional-tests/tests/settings/misc.spec.ts:13:    test.slow(project.name !== 'local', 'email delivery can be slow');
packages/functional-tests/tests/settings/password.spec.ts:16:    test.slow(project.name !== 'local', 'email delivery can be slow');
packages/functional-tests/tests/settings/password.spec.ts:53:    test.slow(project.name !== 'local', 'email delivery can be slow');
```